ceph_deploy/hosts/remotes.py

```diff
@@ -153,6 +153,10 @@ def remove_whitespace_from_assignments():
 def write_conf(cluster, conf, overwrite):
     """ write cluster configuration to /etc/ceph/{cluster}.conf """
     path = '/etc/ceph/{cluster}.conf'.format(cluster=cluster)
+    # Stateless OS like ClearLinux does not have anything populated in /etc.
+    # Create the folder before we write the config file.
+    if not os.path.isdir("/etc/ceph"):
+        os.makedirs("/etc/ceph")
```

Comment:
DANGEROUS! `os.makedirs` defaults to mode 777, change line to `os.makedirs('/etc/ceph', mode=0o700)`

On another hand, try to keep the quotation marks the files is using (single in this case). Would be nice to change both lines.